src/App.js

```diff
@@ -1,7 +1,9 @@
-import Header from '../src/Components/Header/index'
+import Header from '../src/Components/Header/Header'
+import HomePage from './Components/HomePage/HomePage';
 function App() {
   return  <div className="App">
-  <Header></Header>
+  <Header/>
+  <HomePage/>
    </div>
```

Comment:
Format the return like this please:
![image](https://user-images.githubusercontent.com/81507185/232162151-baa225e7-86e0-42e5-8b46-934fa25b3305.png)